### PR TITLE
Duplicates in repo

### PR DESCRIPTION
**Pull request by teizenman**
## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: https://github.com/demisto/etc/issues/23655

## Description
In bug issue

### CODE REVIEW

**Comment by teizenman** [comment]
Build fails due to a test that should be skipped by https://github.com/demisto/content/pull/6600

File: TestPlaybooks/playbook-virusTotalPrivateAPI-test.yml [hunk 1]
```diff
@@ -1,8 +1,8 @@
-id: virusTotal-test-playbook
+id: virusTotalPrivateAPI-test-playbook
```
Comment (ronykoz) [inline TestPlaybooks/playbook-virusTotalPrivateAPI-test.yml:1]: shouldn't we update conf.json too?

File: Packs/Legacy/Widgets/widget-ActiveIncidentsByRole.json [hunk 2]
```diff
@@ -1,5 +1,5 @@
 {
-  "id": "incidents-by-role",
+  "id": "active-incidents-by-role",
```
Comment (ronykoz) [inline Packs/Legacy/Widgets/widget-ActiveIncidentsByRole.json:2]: why are we changing ids

File: Packs/Legacy/Widgets/widget-IndicatorManualNumber.json [hunk 3]
```diff
@@ -1,5 +1,5 @@
 {
-  "id": "indicators-manual-user",
+  "id": "count-indicators-manual-user",
```
Comment (ronykoz) [inline Packs/Legacy/Widgets/widget-IndicatorManualNumber.json:2]: why are we changing ids

File: Packs/Legacy/Playbooks/playbook-PaloAltoNetworks_Cortex_XDR_Incident_Sync.yml [hunk 4]
```diff
@@ -1,234 +0,0 @@
-id: Cortex XDR Incident Sync
```
Comment (ronykoz) [inline Packs/Legacy/Playbooks/playbook-PaloAltoNetworks_Cortex_XDR_Incident_Sync.yml:1]: Why are you removing it

File: Packs/Legacy/Playbooks/playbook-Cortex_XDR_Incident_Handling.yml [hunk 5]
```diff
@@ -1,643 +0,0 @@
-id: Cortex XDR Incident Handling
```
Comment (ronykoz) [inline Packs/Legacy/Playbooks/playbook-Cortex_XDR_Incident_Handling.yml:1]: why are you removing it

**Pull request merged**